src/main/resources/assets/tconstruct/lang/zh_cn.json

```
@@ -0,0 +1,2583 @@
+  "modifier.tconstruct.severing.description": "可以砍掉生物的头部或其他部位！",
+  "modifier.tconstruct.fiery": "炽热",
+  "modifier.tconstruct.fiery.flavor": "注入地狱之火！",
+  "modifier.tconstruct.fiery.description": "点燃目标",
+  "modifier.tconstruct.necrotic": "吸血",
+  "modifier.tconstruct.necrotic.flavor": "吸血鬼！",
+  "modifier.tconstruct.necrotic.description": "击中敌人会消耗他们的生命来治疗你",
+
+  "_comment": "Damage boost",
+  "modifier.tconstruct.piercing": "穿刺",
+  "modifier.tconstruct.piercing.flavor": "变身仙人掌！",
+  "modifier.tconstruct.piercing.description": "工具会造成额外伤害,穿透盔甲攻击！",
+  "modifier.tconstruct.piercing.attack_damage": "穿刺伤害",
+  "modifier.tconstruct.smite": "亡灵杀手",
+  "modifier.tconstruct.smite.flavor": "去死吧,邪恶的生物！",
+  "modifier.tconstruct.smite.description": "亡灵惧怕他的威力！",
+  "modifier.tconstruct.smite.attack_damage": "亡灵伤害",
+  "modifier.tconstruct.bane_of_sssss": "节肢杀手",
+  "modifier.tconstruct.bane_of_sssss.flavor": "你有一个不错的 节肢杀手！",
+  "modifier.tconstruct.bane_of_sssss.description": "工具对蜘蛛、苦力怕和其他令人毛骨悚然的咝咝作响的东西造成额外伤害！",
+  "modifier.tconstruct.bane_of_sssss.attack_damage": "节肢伤害",
+  "modifier.tconstruct.antiaquatic": "海洋杀手",
+  "modifier.tconstruct.antiaquatic.flavor": "基本上是一根钓鱼竿！",
+  "modifier.tconstruct.antiaquatic.description": "工具对海洋生物造成额外伤害！",
+  "modifier.tconstruct.antiaquatic.attack_damage": "海洋生物伤害",
+  "modifier.tconstruct.cooling": "冰之力",
+  "modifier.tconstruct.cooling.flavor": "冷冷的！",
+  "modifier.tconstruct.cooling.description": "工具对火系生物造成额外伤害！",
+  "modifier.tconstruct.cooling.attack_damage": "冷却伤害",
+  "modifier.tconstruct.sweeping_edge": "横扫",
+  "modifier.tconstruct.sweeping_edge.flavor": "武士！",
+  "modifier.tconstruct.sweeping_edge.description": "横扫攻击的目标在此工具的伤害中占较大比例",
+  "modifier.tconstruct.sweeping_edge.attack_damage": "横扫伤害",
+
+  "modifier.tconstruct.sharpness": "锋利",
+  "modifier.tconstruct.sharpness.1": "锋利+",
+  "modifier.tconstruct.sharpness.2": "锋利++",
+  "modifier.tconstruct.sharpness.3": "锋利+++",
+  "modifier.tconstruct.sharpness.4": "锋利++++",
+  "modifier.tconstruct.sharpness.5": "锋利++++",
+  "modifier.tconstruct.sharpness.flavor": "有多锋利？",
+  "modifier.tconstruct.sharpness.description": "工具更加锋利,造成更多伤害！",
+  "modifier.tconstruct.swiftstrike": "迅击",
+  "modifier.tconstruct.swiftstrike.1": "迅击+",
+  "modifier.tconstruct.swiftstrike.2": "迅击++",
+  "modifier.tconstruct.swiftstrike.3": "迅击+++",
+  "modifier.tconstruct.swiftstrike.4": "迅击++++",
+  "modifier.tconstruct.swiftstrike.5": "迅击+++++",
+  "modifier.tconstruct.swiftstrike.flavor": "魔法！",
+  "modifier.tconstruct.swiftstrike.description": "神奇的紫色宝石让武器更轻更容易挥舞！",
+
+  "_comment": "Armor",
+  "modifier.tconstruct.knockback_resistance": "抗击性",
+  "modifier.tconstruct.knockback_resistance.flavor": "一个有分量的主题",
+  "modifier.tconstruct.knockback_resistance.description": "铁砧很重,所以它应该可以防止你被击退,对吧？",
+  "modifier.tconstruct.golden": "金闪闪",
+  "modifier.tconstruct.golden.flavor": "好如金子",
+  "modifier.tconstruct.golden.description": "给你的盔甲额外的金光闪闪,猪灵喜欢它！",
```

```suggestion
  "modifier.tconstruct.knockback_resistance.description": "铁砧很重，所以它应该可以防止你被击退，对吧？",
  "modifier.tconstruct.golden.description": "给你的盔甲额外的金光闪闪，猪灵喜欢它！",
```
中文翻译应使用全角符号。由于存在多处相同问题，建议对全文进行检查和修正。

- **,  -> ，**
- **:  -> ：**